Command/DumpEmoticonsCommand.php

```diff
@@ -68,7 +66,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         if (!file_exists($emoticonsFolder) && !is_dir($emoticonsFolder)) {
-            return $output->writeln('<error>Emoticons folder does not exist</error>');
+            $output->writeln('<error>Emoticons folder does not exist</error>');
```

Comment:
The return is expected here in order to stop the task execution here as the source folder does not exists.

Reply:
The return value inside `execute` should return something useful or nothing at all. Returning the output of `$output->writeln()` is not related to your command (in fact it doesn't even return anything). 

I'll move the return below ``$output->writeln()`.